### DISCUSSION TITLE
Total Balance chart is off

### DISCUSSION THREAD

**Discussion by sc0-0001** [Chia-Network/chia-blockchain#16308] [Support]
The total balance chart seems to be off by roughly a factor of 2. The current balance is correct, but previous balances when I mouse over the line appear to be roughly 2x what I ever had in the wallet.

**Comment by BrandtH22** [comment]
Hey @sc0-0001 , this is a known issue that generally effects users in wallet-only mode moreso than those in farming mode.
The issue occurs because the wallet syncs by polling full nodes and does not always receive the data in the correct order. Syncing from a local full node (farming mode) or with a trusted full node (can set this in the config file) generally resolves the transaction syncing order but the root cause as to why the wallet mode does not realign the graph after sync is achieved is still under review for future GUI optimizations.

Can you confirm what mode the client is in? (wallet or farming)

**Reply by sc0-0001** [reply]
This is indeed in wallet mode.

**Reply by BrandtH22** [reply]
@sc0-0001 ,
Since this is a known bug do you want to:
A. Work through the process to connect to a local or trusted full node and see if this resolves the transaction ordering (not a guaranteed solution).
B. Close this ticket for the time being and add a note to this issue ticket about UI enhancements needed (https://github.com/Chia-Network/chia-blockchain-gui/issues/2123)
C. Something else

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
